tests/sharing/lib/rule.rb

```diff
@@ -25,7 +25,8 @@ def self.create_from_obj(obj, what)
              values: values,
              add: what,
              update: what,
-             remove: what
+             remove: what,
+             object: obj
```

Comment:
What is the purpose of `object`?

Reply:
When playing in `console.rb`, it's sometimes practical to inspect the folder or album associated to a rule.